### PR TITLE
Feedback score

### PR DESCRIPTION
**Pull request by maamalama**

### CODE REVIEW

File: web/components/templates/playground/playgroundPage.tsx [hunk 1]
```diff
@@ -602,7 +602,6 @@ export default PlaygroundPage;
 interface PlaygroundPageProps {
   showNewButton?: boolean;
   request?: string;
-  showNewButton: boolean;
```
Comment (chitalian) [inline web/components/templates/playground/playgroundPage.tsx:605]: why?

File: valhalla/jawn/src/lib/stores/ScoreStore.ts [hunk 2]
```diff
@@ -169,6 +166,92 @@ export class ScoreStore extends BaseStore {
     return ok(rowContents.data);
   }
 
+  public async putScoreIntoClickhouse(newVersion: {
+    id: string;
+    version: number;
+    provider: string;
+    score: Score;
+  }): Promise<Result<InsertRequestResponseVersioned, string>> {
```
Comment (chitalian) [inline valhalla/jawn/src/lib/stores/ScoreStore.ts:174]: Why don't we just reuse the `putScoresIntoClickhouse` function here? We are copying the entire implementation

File: web/components/templates/playground/playgroundPage.tsx (hunk 1 above)
Comment (maamalama) [reply web/components/templates/playground/playgroundPage.tsx:605]: @chitalian Removed duplicate showNewButton from @YashKarthik's PR

File: valhalla/jawn/src/lib/stores/ScoreStore.ts (hunk 2 above)
Comment (maamalama) [reply valhalla/jawn/src/lib/stores/ScoreStore.ts:177]: @chitalian make sense! fixed this

**Review by chitalian — APPROVED** [review]

**Pull request merged**